Common subdirectories: noVNC.orig/app and noVNC/app
Common subdirectories: noVNC.orig/core and noVNC/core
Common subdirectories: noVNC.orig/docs and noVNC/docs
File noVNC.orig/.git is a directory while file noVNC/.git is a regular file
Common subdirectories: noVNC.orig/.github and noVNC/.github
Common subdirectories: noVNC.orig/po and noVNC/po
Common subdirectories: noVNC.orig/snap and noVNC/snap
Only in noVNC.orig: START.sh
Common subdirectories: noVNC.orig/tests and noVNC/tests
Common subdirectories: noVNC.orig/utils and noVNC/utils
Common subdirectories: noVNC.orig/vendor and noVNC/vendor
diff -u noVNC.orig/vnc_lite.html noVNC/vnc_lite.html
--- noVNC.orig/vnc_lite.html	2022-07-19 14:52:07.640528416 -0400
+++ noVNC/vnc_lite.html	2022-11-16 20:49:10.701753363 -0500
@@ -57,11 +57,6 @@
 
     </style>
 
-    <!-- implant to get password from URL -->
-    <script>
-    const queryString = window.location.search;
-    const urlParams = new URLSearchParams(queryString);
-    </script>
     <script type="module" crossorigin="anonymous">
         // RFB holds the API to connect and communicate with a VNC server
         import RFB from './core/rfb.js';
@@ -87,13 +82,7 @@
         // When this function is called, the server requires
         // credentials to authenticate
         function credentialsAreRequired(e) {
-            var password = null;
-            
-            if (urlParams.get('pass') != null) { 
-               password = urlParams.get('pass');
-            } else {
-               password = prompt("Password Required:");
-            }
+            const password = prompt("Password Required:");
             rfb.sendCredentials({ password: password });
         }
 
Only in noVNC.orig: vnc_mini.html
